h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -83,7 +83,7 @@
     public long seed;
 
     // Input fields
-    @API(help = "Family. Use binomial for classification with logistic regression, others are for regression problems.", values = {"gaussian", "binomial","quasibinomial","ordinal", "multinomial", "poisson", "gamma", "tweedie", "negativebinomial"}, level = Level.critical)
+    @API(help = "Family. Use binomial for classification with logistic regression, others are for regression problems.", values = {"gaussian", "binomial","fractionalbinomial","quasibinomial","ordinal", "multinomial", "poisson", "gamma", "tweedie", "negativebinomial"}, level = Level.critical)
```

Comment:
```suggestion
    @API(help = "Family. Use binomial for classification with logistic regression, others are for regression problems.",
            values = {"gaussian", "binomial", "fractionalbinomial", "quasibinomial", "ordinal", "multinomial", "poisson",
                    "gamma", "tweedie", "negativebinomial"}, level = Level.critical)
```

Perfect opportunity to format this block. No other changes made. It looks good in the IDE, not on GitHub.

![Screenshot from 2020-03-23 08-31-03](https://user-images.githubusercontent.com/8769110/77292504-aaaa8f00-6ce0-11ea-9c1c-76f28230988f.png)